app/experimenter/kinto/tests/test_serializers.py

```diff
@@ -0,0 +1,83 @@
+import os
+import json
+from datetime import datetime
+from jsonschema import validate
+from django.test import TestCase
+
+from experimenter.experiments.models import Experiment
+from experimenter.experiments.tests.factories import (
+    ExperimentFactory,
+    ExperimentVariantFactory,
+)
+from experimenter.kinto.serializers import ExperimentRapidRecipeSerializer
+
+
+class TestExperimentRapidSerializer(TestCase):
+    def test_serializer_outputs_expected_schema(self):
+        audience = Experiment.RAPID_AUDIENCE_CHOICES[0][1]
+        features = [feature[0] for feature in Experiment.RAPID_FEATURE_CHOICES]
+        normandy_slug = "experimenter-normandy-slug"
+        experiment = ExperimentFactory.create(
+            type=Experiment.TYPE_RAPID,
+            rapid_type=Experiment.RAPID_AA_CFR,
+            audience=audience,
+            features=features,
+            normandy_slug=normandy_slug,
+            proposed_enrollment=9,
+        )
+
+        ExperimentVariantFactory.create(
+            experiment=experiment, slug="control", is_control=True
+        )
+        ExperimentVariantFactory.create(experiment=experiment, slug="variant-2")
+
+        serializer = ExperimentRapidRecipeSerializer(experiment)
+        data = serializer.data
+
+        fn = os.path.join(os.path.dirname(__file__), "experimentRecipe.json")
+
+        with open(fn, "r") as f:
+            json_schema = json.load(f)
+        self.assertIsNone(validate(instance=data, schema=json_schema))
+
+        arguments = data.pop("arguments")
+        branches = arguments.pop("branches")
+        start_date = arguments.pop("startDate")
+
+        self.maxDiff = None
```

Comment:
We should find a better solution for this, I wish maxDiff was always enabled globally instead of needing to add it and remove it on every test.